examples/gno.land/p/demo/uuid/uuid_test.gno

```diff
@@ -0,0 +1,50 @@
+package uuid
+
+import (
+	"testing"
+)
+
+func TestID(t *testing.T) {
+	u := NewUUID()
+
+	id1 := u.ID()
+	id2 := u.ID()
+	uuid1 := IDToUUID(id1)
+	uuid2 := IDToUUID(id2)
+
+	if uuid1 == uuid2 {
```

Comment:
You can utilize `uassert.Equal` here

Reply:
[91cdfe8](https://github.com/gnolang/gno/pull/2076/commits/91cdfe816c7b98c4453c8e430da9b6fa51723a08)